CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Release History
 
+### 1.7.0
+#### New Features
+* Updated `azure-cosmos` version to 4.41.0.
```

Comment:
Please also call out the upgrade of kafka connect dependency

Reply:
Added

Reply:
never mind - just a test dependency - missed that.